lib/plugins/cannulaage.js

```diff
@@ -3,53 +3,110 @@
 var _ = require('lodash');
 var moment = require('moment');
 
+function cage() {
+  return cage;
+}
+
 function init() {
+  return cage();
+}
+
+module.exports = init;
+
+cage.label = 'Cannula Age';
+cage.pluginType = 'pill-minor';
+
+cage.checkNotifications = function checkNotifications(sbx) {
+
+  if (!sbx.extendedSettings.enablealerts) { return; }
+
+  var cannulaInfo = cage.findLatestTimeChange(sbx);
+
+  if (!cannulaInfo.checkForAlert) { return; }
+
+  // CAGE_INFO = 44 CAGE_WARN=48 CAGE_URGENT=70
+  
+  var info = sbx.extendedSettings.info ? sbx.extendedSettings.info : 44;
+  var warn = sbx.extendedSettings.warn ? sbx.extendedSettings.warn : 48;
+  var urgent = sbx.extendedSettings.urgent ? sbx.extendedSettings.urgent : 72;
+
+  if (cannulaInfo.age === info || cannulaInfo.age === urgent || cannulaInfo.age === warn )
+  {
+    var title = 'Cannula age ' + cannulaInfo.age +' hours';
+    
+    var sound = 'incoming';
+    var msg = ', change due soon';
+    if (cannulaInfo.age === warn) { msg = 'time to change'; }
+    if (cannulaInfo.age === urgent) { msg = 'change overdue!'; sound = 'persistent'; }
+    
+    sbx.notifications.requestNotify({
+      level: 2
+      , title: title
+      , message: title + msg
+      , pushoverSound: sound
+      , plugin: cage
+      , debug: {
+        cannulaInfo: cannulaInfo
+      }
+    });
 
-  function cage() {
-    return cage;
   }
+}
+
+function frac(f) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['frac' is defined but never used.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/769587957/issues/source?bid=2452379&fileBranchId=2457543#l56)